src/plugins/kibana/public/dashboard/styles/main.less

```diff
@@ -3,6 +3,9 @@
 
 .tab-dashboard {
   background-color: @dashboard-bg;
+  navbar[name="dashboard-options"] {
```

Comment:
We should be able to normalize all of the `$appName-options` into a single reusable style